rust/wasmtime_bindings/src/memory_creator.rs

```diff
@@ -0,0 +1,96 @@
+use anyhow::{anyhow, Result};
+use std::{cmp, ptr, slice, u32};
```

Comment:
Copyright/license blurbs needed in all new files.

Reply:
Ok